lib/worker/migrations/20231115184124_cache.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS cache_config
```

Comment:
- creates a new table to store the config for cache. `duration` is meant to be time in minutes
- adds cache_path and expires_at columns to indicates where in cache the file is stored, and when it will be expired